packages/x-charts/src/PieChart/PieArc.tsx

```diff
@@ -66,7 +66,7 @@ const PieArcRoot = styled('path', {
 }));
 
 export type PieArcProps = Omit<PieArcOwnerState, 'isFaded' | 'isHighlighted'> &
-  Omit<React.ComponentPropsWithoutRef<'path'>, 'onClick'> &
+  React.ComponentPropsWithoutRef<'path'> &
```

Comment:
Since we now have the  custom `onClick` set at the `<PiePlot/>` level, we can use native event listener type